policies-procedures-and-docs/communications/hackmd-guide.md

```diff
@@ -0,0 +1,69 @@
+OLS HackMD guide: How to work with this doc?
```

Comment:
@yochannah  Do we still need that as we do not use HackMD anymore?

Reply:
This is the only file which has not been made into a page. I'll wait for you to amswer, Yo...

Reply:
thanks - I think that we CAN use it for mentor/mentee meetings, so yes, let's include it.